src/error/views/ErrorView.tsx

```diff
@@ -45,18 +56,37 @@ const ErrorView: FunctionComponent<ErrorViewProps> = ({
 		message ||
 		i18n.t('error/views/error-view___de-pagina-werd-niet-gevonden');
 	const errorIcon: IconName = queryParams.icon || icon || 'search';
+	const buttons = uniq([
+		...actionButtons,
+		...(queryParams.actionButtons
+			? queryParams.actionButtons.split(',').map(button => button.trim())
+			: []),
+	]);
 
 	return (
 		<Container mode="vertical" background="alt">
 			<Container size="medium" mode="horizontal">
 				<Blankslate body="" icon={errorIcon} title={errorMessage}>
 					{children}
-					{actionButtons.includes('home') && (
-						<Button
-							onClick={() => redirectToClientPage(APP_PATH.LOGGED_IN_HOME, history)}
-							label={t('error/views/error-view___ga-terug-naar-de-homepagina')}
-						/>
-					)}
+					<Toolbar>
+						<ToolbarCenter>
+							<ButtonToolbar>
+								{buttons.includes('home') && (
+									<Button
+										onClick={() => redirectToClientPage(APP_PATH.LOGGED_IN_HOME, history)}
+										label={t('error/views/error-view___ga-terug-naar-de-homepagina')}
+									/>
+								)}
+								{buttons.includes('helpdesk') && (
+									<Button
+										type="danger"
+										onClick={() => (window as any).zE('webWidget', 'toggle')}
```

Comment:
Je kan die toevoegen aan de Window interface in `shared/types/global.d.ts`, dan moet die as any niet.